tools/development/count_test_cases.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+##
+# GTest / SSAT Test Result Aggregator
+# Copyright (c) 2019 Samsung Electronics
+#
+# You may use this under either LGPL 2.1+ or Apache-2.0.
+
+##
+# @file   count_test_cases.py
+# @brief  A unit test result aggregator
+# @author MyungJoo Ham <myungjoo.ham@samsung.com>
+# @date   18 Dec 2019
+# @bug    No known bugs
+# @todo   WIP
+
+import sys
+import re
+import os
+
+## @brief read results of GTest's result xml file
+def readGtestXml(filename):
+    f = open(filename, "r")
```

Comment:
recommended `with open() as f:` for auto close

Reply:
Fixed.